addons/website_partner/models/res_partner.py

```diff
@@ -9,7 +9,7 @@ class WebsiteResPartner(models.Model):
     _name = 'res.partner'
     _inherit = ['res.partner', 'website.seo.metadata']
 
-    website_description = fields.Html('Website Partner Full Description', strip_style=True, translate=html_translate)
+    website_description = fields.Html('Website Partner Full Description', sanitize_overridable=True, translate=html_translate)
```

Comment:
@bvr-odoo I don't think `strip_style` should be removed? Given my explanation https://github.com/odoo/odoo/pull/165482#issuecomment-2135528653 I still think that this is to be sanitized fully for non admin users?